extra_data/reader.py

```diff
@@ -1824,6 +1834,14 @@ def open_run(
         Enable or disable opening files in parallel. Particularly useful if
         creating child processes is not allowed (e.g. in a daemonized
         :class:`multiprocessing.Process`).
+    aliases: str, Path, dict
```

Comment:
I think I would indeed prefer to only supports paths (a single one actually!) here. In the rare case where someone actually wants to manually define aliases here, they can always follow up with `.with_aliases()`. It might also make the docstring more clear.

Reply:
Yep, makes sense :+1: Fixed in 4794482.